### ISSUE TITLE
Is MiMa really working for 2.12.1?

### ISSUE THREAD

**Issue by retronym** [scala/scala-dev#264]
See: https://github.com/scala/scala/pull/5528

**Comment by retronym** [comment]
This shows that changing a private method to a protected method is not forwards compatible (a client compiled against 2.12.1 incurs a linkage error with 2.12.0)

```
⚡ tail /tmp/v{1,2}.scala
==> /tmp/v1.scala <==
  protected def foo = 42 // 2.12.1
}
class U extends T {
  def test = foo
}
object Test {
  def main(args: Array[String]): Unit = {
    new U().test
  }
}
==> /tmp/v2.scala <==
trait T {
  private def foo = 42 // 2.12.0
}
⚡ scalac /tmp/v1.scala && scalac /tmp/v2.scala && scala Test
java.lang.NoSuchMethodError: T.foo$(LT;)I
    at U.foo(v1.scala:4)
    at U.test(v1.scala:5)
    at Test$.main(v1.scala:9)
```

**Comment by retronym** [comment]
Eek! MiMa has been in  🙉 🙈 🙊 mode for a while. Fixed in #138.
